src/components/application_manager/src/mobile_command_factory.cc

```diff
@@ -450,6 +452,17 @@ CommandSharedPtr MobileCommandFactory::CreateCommand(
       }
       break;
     }
+    case mobile_apis::FunctionID::GetSystemCapabilityID: {
+      if ((*message)[strings::params][strings::message_type] ==
+          static_cast<int>(application_manager::MessageType::kResponse)) {
+        command = utils::MakeShared<commands::GetSystemCapabilityResponse>(
+            message, application_manager);
+      } else {
+        command = utils::MakeShared<commands::GetSystemCapabilityRequest>(
```

Comment:
Other commands do `command.reset(...)`, do you know whether this will cause issues?